pkg/module/metrics/latency.go

```diff
@@ -12,7 +12,7 @@ import (
 	"time"
 
 	"github.com/cilium/cilium/api/v1/flow"
-	v1 "github.com/cilium/cilium/api/v1/flow"
+	
```

Comment:
the linter will not like this empty line, it should be removed

Reply:
Also there are references of this v1 package name which needs to be corrected as well.